packages/feeds-client/__integration-tests__/activity-websocket-events.test.ts

```diff
@@ -64,6 +64,11 @@ describe('Activity state updates via WebSocket events', () => {
     expect(feedActivities?.length).toBe(1);
     expect(feedActivities?.[0].id).toBe(activity.id);
     expect(feedActivities?.[0].text).toBe('Test activity');
+
+    // Verify that the cache is updated
+    const indexedActivityIds = (feed as any).indexedActivityIds;
+    expect(indexedActivityIds.size).toEqual(1);
+    expect(indexedActivityIds.has(activity.id)).toBe(true);
```

Comment:
```suggestion
    const indexedActivityIds = (feed as any).indexedActivityIds;
    expect(indexedActivityIds.size).toEqual(1);
    expect(indexedActivityIds.has(activity.id)).toBe(true);
```
Maybe let's use the actual public API where we can rather than the internals, this tends to create problems down the road when we decide to change these internals (obviously not in this case but still).